rllib/utils/exploration/epsilon_greedy.py

```diff
@@ -153,33 +148,5 @@ def _get_torch_exploration_action(self, q_values, explore, timestep):
 
     @override(Exploration)
     def get_info(self):
-        """Returns the current epsilon value.
-
-        Returns:
-            Union[float,tf.Tensor[float]]: The current epsilon value.
-        """
-        return self.epsilon_schedule(self.last_timestep)
-
-    @override(Exploration)
-    def get_state(self):
-        return [self.last_timestep]
-
-    @override(Exploration)
-    def set_state(self, state):
-        if self.framework == "tf" and tf.executing_eagerly() is False:
-            update_op = tf.assign(self.last_timestep, state)
-            with tf.control_dependencies([update_op]):
-                return tf.no_op()
-        self.last_timestep = state
-
-    @override(Exploration)
-    def reset_state(self):
-        return self.set_state(0)
-
-    @classmethod
-    @override(Exploration)
-    def merge_states(cls, exploration_objects):
-        timesteps = [e.get_state() for e in exploration_objects]
-        if exploration_objects[0].framework == "tf":
-            return tf.reduce_sum(timesteps)
-        return np.sum(timesteps)
+        eps = self.epsilon_schedule(self.last_timestep)
+        return {"cur_epsilon": eps}
```

Comment:
I looked into trying to avoid making get_info() return TensorType in graph mode, but that seemed like a big change so punting on that.